pages/agent/v3/help/_artifact_shasum.txt

```diff
@@ -16,7 +16,7 @@ Example:
    $ buildkite-agent artifact shasum "pkg/release.tar.gz" --build xxx
 
    This will search for all the files in the build with the path "pkg/release.tar.gz" and will
-   print to STDOUT it's SHA-1 checksum.
+   print to STDOUT its SHA-1 checksum.
```

Comment:
As above.

Also, these files in the `help` directory are generated from https://github.com/buildkite/agent/, please open a PR there to fix the source if you'd like to.

Reply:
Thanks for the pointer, @plaindocs. I took a look at buildkite/agent and it looks like there was a "it's" -> "its" change made 5 months ago in https://github.com/buildkite/agent/commit/a52af04aa69f8d844996842224f5aaa8c358ff93

Do you know if there's a process to regenerate the docs in buildkite/docs from buildkite/agents? It seems like the source was changed but it didn't actually propagate to this repo. Otherwise, happy to make the changes in both places.

Reply:
Hey @vineetgopal that would be great. I'm looking into a process for making sure changes propagate.

Reply:
Posted a change here: https://github.com/buildkite/agent/pull/1229